exporter/elasticsearchexporter/model.go

```diff
@@ -480,6 +481,52 @@ func summaryToValue(dp pmetric.SummaryDataPoint) pcommon.Value {
 	return vm
 }
 
+// exponentialHistogramToValue converts an exponential histogram data point to T-digest.
+func exponentialHistogramToValue(dp pmetric.ExponentialHistogramDataPoint) (pcommon.Value, error) {
+	vm := pcommon.NewValueMap()
+	m := vm.Map()
+	counts := m.PutEmptySlice("counts")
+	values := m.PutEmptySlice("values")
+
+	scale := int(dp.Scale())
+
+	offset := int(dp.Negative().Offset())
+	bucketCounts := dp.Negative().BucketCounts()
+	for i := bucketCounts.Len() - 1; i >= 0; i++ {
+		count := bucketCounts.At(i)
+		if count == 0 {
+			continue
+		}
+		lb := -exphistogram.LowerBoundary(offset+i+1, scale)
+		ub := -exphistogram.LowerBoundary(offset+i, scale)
+		counts.AppendEmpty().SetInt(int64(count))
+		values.AppendEmpty().SetDouble(lb + (ub-lb)/2)
+	}
+
+	if zeroCount := dp.ZeroCount(); zeroCount != 0 {
+		counts.AppendEmpty().SetInt(int64(zeroCount))
+		// FIXME: do we really have to take the median of last negative and first positive, instead of using 0?
```

Comment:
I'd expect us to be able to just use `0`. If the offsets if the positive and negative scale are the same, the midpoint is always zero. If the offsets are different, I don't think the midpoint value is meaningful. As the zero count is the count of values that are zero (or within the `zero_threshold`), I think we should just use `0` as the value`.